### PR TITLE
fixed that `CLONE` did not set `x` or `y` to 0

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
It would instead fallback to the `x` and `y` of the cloned widget.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3654/pr-test (or any other room on that server)

**Comment by bjalder26** [comment]
If fixed that issue, but now if I set a clone to "x":40 in CLONE/properties it will appear in the right place, show NaN in the JSON editor, then snap to "x":0 when selected.

After cloned, but before selected (appears at "x":40):
image

After selected:
image

File: client/js/widgets/widget.js [hunk 1]
```diff
@@ -590,8 +590,8 @@ export class Widget extends StateManaged {
 
               // moveToHolder causes the position to be wrong if the target holder does not have alignChildren
               if(!parent || !widgets.get(parent).get('alignChildren')) {
-                await cWidget.set('x', (a.properties.x || w.get('x')) + a.xOffset * i);
-                await cWidget.set('y', (a.properties.y || w.get('y')) + a.yOffset * i);
+                await cWidget.set('x', (a.properties.x !== undefined ? a.properties.x : w.get('x')) + a.xOffset * i);
+                await cWidget.set('y', (a.properties.y !== undefined ? a.properties.x : w.get('y')) + a.yOffset * i);
```
Comment (rogerl50) [inline client/js/widgets/widget.js:594]: `a.properties.x` s/b `a.properties.y`

**Review by rogerl50 — CHANGES_REQUESTED** [review]

**Comment by bjalder26** [comment]
Somehow it is now multiplying y values by 10.  So if I use: 
    {
      "func": "CLONE",
      "source": "thisButton",
      "properties": {
        "y": "50"
      }
    }

It will wind up with a y value of 500 (40 => 400).

However, when there is a decimal, it just adds a zero to the end of the decimal (e.g. 12.3 => 12.30).

**Comment by ArnoldSmith86** [comment]
Kind of expected when you provide `"50"` instead of `50`. Don't know if that should be changed.

File: client/js/widgets/widget.js (hunk 1 above)
Comment (bjalder26) [reply client/js/widgets/widget.js]: > Kind of expected when you provide `"50"` instead of `50`. Don't know if that should be changed.

My mistake.

**Review by rogerl50 — APPROVED** [review]

**Review by bjalder26 — APPROVED** [review]
Can't find any issues, and it fixes a known issue.  Approved through testing.

**Pull request merged**